app/assets/stylesheets/master.scss

```diff
@@ -236,4 +235,18 @@ p {
 
 .modal-backdrop {
   z-index: 1030;
-}
\ No newline at end of file
+}
+
+.dashboard{
+  background-color: #D1D1D1;
+  min-height: 200px;
+  padding: 30px;
+}
+.user-photo{
+  border-radius: 50%;
+  border: 5px solid #AAAAAA;
+  height: 150px;
+  width: 150px;
+  background:url(trump.jpg) center;
```

Comment:
Ah, that makes sense.  This is related to the avatar uploading, but it's only fleshing out the feature, so you still have the ticket to work on, @NKHolton, but it might make sense to work with @cacauzen on image uploading since she was working on this code which is related.